Source/Api/Controllers/EventController.cs

```diff
@@ -625,6 +625,15 @@ public class EventController : RepositoryApiController<IEventRepository, Persist
 
             return Ok();
         }
+        
+        [HttpPost]
+        [Route("~/entries")]
```

Comment:
Haven't tried `raygun` myself - just expected one to be able to pass to a url specified by the user, like webhooks on GitHub and other service providers.

Yes, I can try help @skoub out.